client/src/cb-admin/pages/Settings.js

```diff
@@ -157,11 +157,11 @@ export default class SettingsPage extends React.Component {
   updateStateFromApi = (data) => {
     this.setState({
       id: data.id,
-      orgName: data.org_name,
-      sector: data.genre,
-      email: data.email,
+      orgName: data.name,
+      sector: data.sector,
+      email: data.email, // not returned in response
```

Comment:
The CB email isn't in the response from the API? Is that because you're hitting `/organisations/me` instead of `/community-businesses/me`?

Reply:
@eliascodes according to the `api.json` community-business is an alias for organisations. If this is no longer the case can you please make an issue so I know what changes are being made.

Reply:
Sorry, I forgot that `email` is part of the user model now and will need a separate call. Your response stands, we'll talk about that elsewhere.

Reply:
So are we going to do the call to get the `email` in a different PR or in this one?

Reply:
@eliascodes are we enforcing one admin account per community business? If so, this can be dealt with in this PR, if not we need to have a think about how we are relating email address to cbs.

Reply:
> are we enforcing one admin account per community business?

For now, I think so

Reply:
If we're enforcing 1 admin account per cb, do we want to add the email address to the cb model?

Or hit the `/community-business/me/users/admins` endpoint?

Reply:
Yeah, that makes sense. Technically, then we do not need to enforce 1 admin per cb.